src/screens/InputUsername/index.js

```diff
@@ -309,9 +310,9 @@ const ChooseUsername = () => {
                 {messageTypeFetch(typeFetch, formatUsernameString())}
               </View>
             </View>
-            <View style={styles.constainerInfo}>
+            <View style={styles.containerInfo}>
               <View style={styles.parentIcon}>
-                <WarningIcon />
+                <IconWarningCircleBlue fill={COLORS.white2} fillIcon={COLORS.signed_primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fill` and `fillIcon` props have been added to the `IconWarningCircleBlue` component. Ensure these changes don't affect the component's behavior and appearance negatively.
```diff
- <WarningIcon />
+ <IconWarningCircleBlue fill={COLORS.white2} fillIcon={COLORS.signed_primary} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->